code/game/gamemodes/malfunction/Malf_Modules.dm

```diff
@@ -215,16 +191,22 @@ rcd light flash thingy on matter drain
 	user_type = USER_TYPE_MALFAI
 	panel = MALFUNCTION
 	charge_type = Sp_CHARGES
-	charge_max = 3
+	charge_max = 1
 	hud_state = "blackout"
 	override_base = "grey"
 
 /spell/aoe_turf/blackout/cast(var/list/targets, mob/user)
 	for(var/obj/machinery/power/apc/apc in power_machines)
-		if(prob(30*apc.overload))
-			apc.overload_lighting()
-		else
-			apc.overload++
+		apc.overload_lighting()
+	to_chat(world, radio_list.len)
+	for(var/obj/item/device/radio/R in radio_list)
+		R.scramble_message = 1
+	for(var/obj/item/device/rcd/matter/engineering/rcd in rcd_list)
```

Comment:
Honestly just make RCDs check and break themselves instead, the way it is done currently is pretty much just shitcode(not your fault).